WooCommerce/Classes/ViewRelated/Search/SearchViewController.swift

```diff
@@ -179,7 +184,16 @@ where Cell.SearchModel == Command.CellViewModel {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
         let model = resultsController.object(at: indexPath)
-        searchUICommand.didSelectSearchResult(model: model, from: self)
+        searchUICommand.didSelectSearchResult(model: model, from: self, reloadData: { [weak self] in
+            self?.tableView.reloadData()
+            }, updateActionButton: { [weak self] in
```

Comment:
Is this over indented? 🧐

Reply:
hmm I'm not super sure what's the best way to auto format the code with the desirable indentation in this case 🤔  I usually auto format the code in Xcode by ctrl+i, and this indentation is a result of that (same if I delete the code and hit enter from the previous line).

in any case, I manually fixed the indentation here: https://github.com/woocommerce/woocommerce-ios/pull/2541/commits/977a21a5b6eb42bbe13e76d6aad55af19b67774e